bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/utils/Utils.java

```diff
@@ -273,4 +277,151 @@ public static Throwable findRootCause(Throwable t) {
 
         return cause;
     }
+
+	/**
+	 * Enable or disable app monitoring by adding or removing app monitoring
+	 * configuration in the target folder.
+	 * 
+	 * 
+	 * @param isEnable flag used to enable or disable app monitoring..
+	 * @param project  a project in the Liberty dashboard.
+	 * 
+	 */
+	public static void enableAppMonitoring(Boolean isEnable, Project project) {
+
+		if (project == null || project.getPath() == null || project.getBuildType() == Project.BuildType.UNKNOWN) {
+			System.err.println("Invalid project object or path.");
+			return;
+		}
+
+		if (isEnable) {
+			addConfigDropinsInServerDir(project);
+		} else {
+			try {
+				String dirNameTofind = "configDropins";
+
+				String serverDirName = project.getBuildType() == Project.BuildType.MAVEN ? "target" : "build";
+				File usrDir = new File(project.getPath() + "/" + serverDirName);
+				File configDropins = findFolder(usrDir, dirNameTofind);
+
+				// Delete the directory if exists.
+				if (configDropins != null) {
+					deleteDirectory(configDropins);
+				} else {
+					System.err.println("configDropins directory not found!!");
+				}
+
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+		}
+	}
+
+	// Add configuration to disable app monitoring in target folder.
+	private static void addConfigDropinsInServerDir(Project project) {
+		String serverFolderName = "servers";
+		String fileNameTofind = "server.xml";
+		String usrDirPath = "/liberty/wlp/usr";
+		String newFilePath = "/configDropins/overrides/disableApplicationMonitor.xml";
+		String fileContent = "<server> <applicationMonitor updateTrigger=\"disabled\"/> </server>";
+
+		try {
+			String serverDirName = project.getBuildType() == Project.BuildType.MAVEN ? "target" : "build";
+			File projectDir = new File(project.getPath() + "/" + serverDirName + usrDirPath);
```

Comment:
Same as comment above - we should use Paths.get to create directory paths as it will account for the OS path separators.